openwisp_monitoring/device/static/monitoring/js/chart.css

```
@@ -0,0 +1,156 @@
+  color: var(--ow-overlay-60) !important;
+  font-size: 13px;
+  font-weight: bold;
+}
+#ow-chart-contents b {
+  background-color: var(--ow-color-black) !important;
+}
+.chart-heading {
+  margin: 0;
+  font-size: 2em;
+  text-align: center;
+  font-size: 1.22em;
```

Biome (2.4.7)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)